server_node/files/dummy-web-server.js

```diff
@@ -40,14 +40,16 @@ function process(pool, req, res) {
         			return;
       			}  
   
-      			connection.query("insert into reto1 values (?, now())", params.value, function(err, result) {
-        			connection.release();
-  
+      			connection.query("insert into reto1 values (?, current_timestamp(6))", params.value, function(err, result) {
         			if (err) {
           				console.log(err);
+        				if (connection !== undefined) {
+          					connection.release(); 
+        				}
           				return;
         			} 
-  
+          			connection.release(); 
```

Comment:
No se si está mál. Ahora no puedo mirarlo. Le echarré un ojo mas tarde :-)

On Mon, May 23, 2016 at 9:20 AM, Juan Arias <notifications@github.com>
wrote:

> In server_node/files/dummy-web-server.js
> <https://github.com/misterarias/RETO2/pull/12#discussion_r64177710>:
>
> >            				return;
> >          			}
> > -
> > +          			connection.release();
>
> Si ha habido un error en la query, la conexión se liberaría dos veces,
> esto no podría ser un error?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/misterarias/RETO2/pull/12/files/6d361d06769a1c7cf7437837426c021672cbf123#r64177710>
>